Documentation/gettingstarted.rst

```diff
@@ -697,3 +698,236 @@ If instead you just want to shut down the VM but may use it later,
+    ==> default: Installing new /host/etc/cni/net.d/10-cilium.conf ...
+  
+    $
+
+If the script exits with an error message, do not attempt to proceed with the
+tutorial, as later steps will not work properly.   Instead, contact us on the
+`Cilium Slack channel <https://cilium.herokuapp.com>`_.
+
+Step 3: Accessing the VM
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+After the script has successfully completed, you can log into the VM using
+``vagrant ssh``:
+
+::
+
+    $ vagrant ssh
+
+
+All commands for the rest of the tutorial below should be run from inside this
+Vagrant VM.  If you end up disconnecting from this VM, you can always reconnect
+in a new terminal window just by running ``vagrant ssh`` again from the Cilium
+directory.
+
+Step 4: Confirm that Cilium is Running
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Cilium agent is now running and you can interact with it using the ``cilium`` CLI client. Check the status of the agent by running ``cilium status``:
+
+::
+
+    $ cilium status
+    KVStore:            Ok
+    ContainerRuntime:   Ok
+    <...>
+    Cilium:             Ok
+
+The status indicates that all necessary components are operational.
+
+Step 5: Run Script to Start Marathon
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start Marathon inside of the vbox:
+
+::
+
+    $ ./start_marathon.sh
+
+Step 6: Start a Simple Web-Server and Client 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The web-server simply responds to requests on a particular port. Use the ``curl`` command below to launch it.
+
+::
+
+    $ curl -i -H 'Content-Type: application/json' -d @web-server.json 127.0.0.1:8080/v2/apps
+
+Confirm that Cilium sees the new workload. The output should return the endpoint with label ``mesos:id=web-server`` and the assigned IP:
+
+::    
+
+    $ cilium endpoint list 
+
+Use the IP from the previous output to test the web-server:
+
+::    
+
+    $ curl <IP>:8181/api 
+
+Run a client that makes requests to the simple server. The script should generate a ``client.json`` for the client task:
+
+::
+
+    $ ./generate_client_file.sh 
+
+Then submit the client task, which will generate ``GET /public`` and ``GET /private`` requests:
+
+::
+
+    $ curl -i -H 'Content-Type: application/json' -d @client.json 127.0.0.1:8080/v2/apps
+
+You can observe the newly created endpoint in Cilium, similar to the following output:
+
+::
+
+    $ cilium endpoint list
+    ENDPOINT   POLICY        IDENTITY   LABELS (source:key[=value])   IPv6                   IPv4           STATUS   
+               ENFORCEMENT                                                                                           
+    29898      Disabled      256        mesos:id=web-server           f00d::a00:20f:0:74ca   10.15.242.54   ready    
+    33115      Disabled      257        mesos:id=client               f00d::a00:20f:0:815b   10.15.220.6    ready
+
+There is no easy way to retrieve the logs of the client, so use the following script:
```

Comment:
I would rephrase this so it doesn't sound like Cilium is at fault for not providing easier log access. I would write something like this:

Marathon runs all tasks as batch jobs and makes the stdout output available via a log file in `/var/lib/mesos`. The `tail_client.sh` script will identify the log file and then run `tail` on it.